addons/website_slides/controllers/mail.py

```diff
@@ -42,8 +44,13 @@ def mail_update_message(self, thread_model, thread_id, message_id, post_data, **
             raise Forbidden()
         # fetch and update mail.message
         message_id = int(message_id)
-        message_body = plaintext2html(post_data.get('body', ''))
-        subtype_comment_id = request.env['ir.model.data']._xmlid_to_res_id('mail.mt_comment')
+        body_html_string = str(plaintext2html(post_data.get("body", "")))
+        tree = html.fragment_fromstring(body_html_string, create_parent="div")
+        last_p_element = tree.xpath("//p[last()]")[0]
+        last_p_element.text = (last_p_element.text or "") + " "
+        etree.SubElement(last_p_element, "span", attrib={"class": "o-mail-Message-edited"})
+        message_body = Markup("".join(etree.tostring(child, encoding="unicode") for child in tree))
+        subtype_comment_id = request.env["ir.model.data"]._xmlid_to_res_id("mail.mt_comment")
```

Comment:
This lacks explanation in code and/or commit/PR message and/or comment here to explain what this does, this is quite low-level and I wonder what this does.

I think this is to remove `.o-mail-Message-edited` from the html?
Couldn't we just prevent putting it in JS?
Also what's the problem with having this in the body? Code to determine whether message is empty or not works regardless of presence of `o-mail-Message-edited`, no?

Reply:
I'm not super confident with this code relying on the opinionated working of the html_sanitizer, but at least this code is specific to v18 so that's alright.

@seb-odoo I let you confirm whether this is ok to have this fix merged.

Reply:
We are going to have the same issue in 19.0 with the html editor no?

The question is basically to determine where in the DOM we put the edited tag.

In the current case, can you remind me what happens if we place it after the result of `plaintext2html` instead of inside its last `p`?

Reply:
I'll wait the answer to this for the final decision on how we should proceed for this part.